content/cli/overview.md

```diff
@@ -2,6 +2,13 @@
 
 The [Nest CLI](https://github.com/nestjs/nest-cli) is a command-line interface tool that helps you to initialize, develop, and maintain your Nest applications. It assists in multiple ways, including scaffolding the project, serving it in development mode, and building and bundling the application for production distribution. It embodies best-practice architectural patterns to encourage well-structured apps.
 
+#### Requirements
+
+Nest CLI requires a Node.js binary built with [internationalization support](https://nodejs.org/api/intl.html) (ICU), such as the official binaries from the [Node.js project page](https://nodejs.org/en/download). If you encounter errors related to ICU, check that your binary meets this requirement.
+
+```bash
+node -p process.versions.icu
+
```

Comment:
```suggestion
```bash

If the command prints `undefined`, your Node.js binary has no internationalization support.

```